app/models/ems_event.rb

```diff
@@ -196,6 +196,7 @@ def get_target(target_type)
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
     target_type = "middleware_server"   if event.event_type == "hawkular_alert"
+    target_type = "container_node"      if event.event_type == "datawarehouse_event"
```

Comment:
Sufficient now but you need a plan for deriving target_type when you'll have more entities beside container_node...